jaba/src/pages/ForgotPassword/ForgotPasswordFrontEnd.module.css

```diff
@@ -0,0 +1,86 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@300&display=swap');
```

Comment:
Can you see if you can rebase your branch with main and use the custom font files that were added to the repo instead of importing with the url?